bin/logstash.bat

```diff
@@ -81,9 +81,9 @@ goto :end
 
 :concat
 IF not defined CLASSPATH (
-  set CLASSPATH="%~1"
+  set CLASSPATH=%~1
```

Comment:
there were double quotes wrapping up all of CLASSPATH, causing the issue we see when starting `bin\logstash.bat`. I don't see any warm in removing them